generated/src/aws-cpp-sdk-s3-crt/source/S3CrtClient.cpp

```diff
@@ -480,6 +492,7 @@ void S3CrtClient::init(const S3Crt::ClientConfiguration& config,
   {
     aws_tls_connection_options_clean_up(&nonConstTlsOptions);
   }
+  aws_retry_strategy_release(s3CrtConfig.retry_strategy);
```

Comment:
`aws_retry_strategy_release` does nothing when given a `NULL` parameter, so safe to use with defaults.